pallets/summary/src/lib.rs

```diff
@@ -131,8 +132,15 @@ pub mod pallet {
         type BridgeInterface: avn::BridgeInterface;
         /// A flag to determine if summaries will be automatically sent to Ethereum
         type AutoSubmitSummaries: Get<bool>;
+        /// A flag to determine if watchtower validation is required before submitting/anchoring summaries
+        #[pallet::constant]
+        type RequireWatchtowerValidation: Get<bool>;
```

Comment:
nit: do we want to reference the pallet name or keep it generic? Example: `RequireExternalValidation` vs `RequireWatchtowerValidation`